ui/raidboss/data/03-hw/dungeon/gubal_library_hard.js

```diff
@@ -110,7 +110,7 @@
       infoText: {
         en: 'Stand in red',
         de: 'Im Roten stehen',
-        fr: 'Restez dans le rouge',
+        fr: 'Tenez-vous dans le rouge',
```

Comment:
Hi there,
You overthink too much. In my opinion the old sentence was perfectly correct and matches the en meaning.

These are old content, I agree that we need to be precise, but not on this.

Reply:
Tu préfères que je rétablisse l'ancienne trad ?

Reply:
Non pas forcément que tu reviennes à l'ancienne version, mais sur des contenus HW on s'en cogne un peu (et y'a peu de risque de crever vu le stuff actuel).

Mon avis est que sur les contenus antérieurs à ShB, ça ne sert à rien de modifier la traduction, sauf si elle n'existe pas.

Concernant tes questions :
1/ Ça revient à la même chose "Restez" / "Tenez-vous dans" c'est pareil. Tant que c'est pas mélangé dans la même instance, osef
2/ Le mot français c'est étourdir, en effet. Mais étourdissez c'est long, c'est pour ça que mes prédécesseurs et moi on était parti sur stun.
3/ Oui ça peut éviter la confusion pour les healers. Mais bon normalement ils sont censé savoir que si le debuff n'a pas de ligne blanche, ils ne peuvent pas le guérir ... Enfin normalement :relaxed:

Reply:
Ça me dérange pas de le faire, même si dans le fond tu as raison. Je le fais pour me dire, que je n'ai rien laissé de côté. Tu le sais, je suis maniaque. Cela m'est pénible par moment.
Je vais laisser ou corriger pour avoir Stun, je le préfère aussi. Mais le silence lui je l'aimait plus depuis leur changement.
Si tu es ok, je corrigerai le reste pour Purifiez. Et l'histoire du trait sur le debuff, ou la ligne qui brille pour le silence, ce n'est que depuis Shb. J'attend ta réponse avant tou changement.
Merci de ton aide.

Reply:
Le trait blanc sur le debuff ça existe depuis ARR, le reste oui c'est nouveau.
Oui pour _Purifier_, sauf dans le cas ou il faut utiliser le sort Guérison (Esuna) dans ce cas il vaut mieux garder _Guérir_.

Reply:
Je laisserai Guérison pour ceux la, car il y a certains debuff ou il faut soignez pour l'enlever, et cela porte à confusion, je trouve. Merci des tes réponses.

Reply:
Bon, y a d'autre endroits où c'est dissipez.. Lequel tu préfère ? Dissipez ou Purifiez (pour la 
 mécanique solo).
Pour le heal on laisse Guérison.
Désolé pour la prise de tête.

Reply:
Si y'a déjà Dissipez autant partir la dessus, ça évitera d'autres modifications

Reply:
C'est moi qui avais traduit par Dissipez, j'avais zap. Merci moon.

Reply:
C'est toi qui à traduit les tea alex, bahamut, ultima ?